h5p-bildetema/src/components/LanguageSelector/LanguageSelector.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { LanguageSelector } from "./LanguageSelector";
+
+export default {
+  title: "Components/LanguageSelector",
+  component: LanguageSelector,
+} as ComponentMeta<typeof LanguageSelector>;
+
+const Template: ComponentStory<typeof LanguageSelector> = args => (
+  // eslint-disable-next-line react/jsx-props-no-spreading
+  <LanguageSelector {...args} />
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
+  languages: [
+    {
+      label: "Norsk",
+      code: "no",
+      rtl: false,
+    },
+    {
+      label: "Engelsk",
+      code: "en",
+      rtl: false,
+    },
+    {
+      label: "Dansk",
+      code: "dk",
+      rtl: false,
+    },
```

Comment:
I think maybe we should add one of the right-to-left languages as an example. Also a language with a really long name, to test out the different cases.